drv/transceivers-server/src/udp.rs

```diff
@@ -833,4 +878,23 @@ impl ServerImpl {
         ));
         result.chain(self.wait_and_check_i2c(result.success()))
     }
+
+    fn get_led_state_response(
+        &mut self,
+        modules: LogicalPortMask,
+        out: &mut [u8],
+    ) -> (usize, ModuleResultNoFailure) {
+        let mut led_state_len = 0;
+        for led_state in modules.to_indices().map(|m| self.get_led_state(m)) {
+            let led_state_size =
+                hubpack::serialize(&mut out[led_state_len..], &led_state)
```

Comment:
Do we check elsewhere to make sure it's impossible to overflow the buffer here?  If so, add a comment explaining that's why the `unwrap` is okay.